code/modules/reagents/chemistry/holder.dm

```diff
@@ -584,7 +584,7 @@
 			can_process = TRUE
 	return can_process
 
-/datum/reagents/proc/reaction(atom/A, method = TOUCH, volume_modifier = 1, show_message = 1)
+/datum/reagents/proc/reaction(atom/A, method = TOUCH, volume_modifier = 1, show_message = 1, obj/item/bodypart/affecting)
 	var/react_type
```

Comment:
Unrelated changes

Reply:
Related to patches caring about which body part you're targeting to do their effects.